hypervisor/arch/x86/guest/vmcs.c

```diff
@@ -531,7 +549,7 @@ static void init_exit_ctrl(const struct acrn_vcpu *vcpu)
 	value32 = check_vmx_ctrl(MSR_IA32_VMX_EXIT_CTLS,
 			 VMX_EXIT_CTLS_ACK_IRQ | VMX_EXIT_CTLS_SAVE_PAT |
 			 VMX_EXIT_CTLS_LOAD_PAT | VMX_EXIT_CTLS_LOAD_EFER |
-			 VMX_EXIT_CTLS_SAVE_EFER | VMX_EXIT_CTLS_HOST_ADDR64);
```

Comment:
WHy we need to LOAD_EFER?

Reply:
> WHy we need to LOAD_EFER?

It's just the initial value, and the LOAD_EFER bits will be removed later if EFER is identical between guest and host.

Anyway, I can remove it, less code.